drop-transfer/src/ws/server/mod.rs

```diff
@@ -166,14 +167,25 @@ pub(crate) fn spawn(
                     let stop = stop.clone();
                     let logger = logger.clone();
                     let nonces = nonces.clone();
+                    // TODO: why does this help
+                    let refresh_trigger = refresh_trigger.clone();
```

Comment:
Because you have `async move` down the line, so it moves the copy. Without it, you try to move multiple times the same variable

Reply:
thanks